NitroxModel-Subnautica/Packets/CyclopsChangeSonarMode.cs

```diff
@@ -4,25 +4,25 @@
 namespace NitroxModel_Subnautica.Packets
 {
     [Serializable]
-    public class CyclopsActivateSonar : Packet
+    public class CyclopsChangeSonarMode : Packet
     {
         public string Guid { get; }
-        public bool Active { get; }
+        public bool IsOn { get; }
 
-        public CyclopsActivateSonar(string guid)
+        public CyclopsChangeSonarMode(string guid)
         {
             Guid = guid;
-            Active = false;
+            IsOn = false;
```

Comment:
minor: Should we remove this constructor to force people to make a decision?

Reply:
Yes we should.